build/tasks/CreateCommonManifestTask.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json.Linq;
+
+namespace RepoTasks
+{
+    public class CreateCommonManifestTask : Task
```

Comment:
Can you add a description of what this task does (for posterity and the next guy who has to edit this)?